app/src/main/java/com/example/drawingApp/MainViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.drawingApp
+
+import androidx.annotation.ColorInt
+import androidx.lifecycle.ViewModel
+
+class MainViewModel : ViewModel() {
+
+
+    @ColorInt
+    var primaryColor = 0
```

Comment:
0 is Color.TRANSPARENT

Reply:
I suppose I should have it as black since that's the standard starting Color, I'll change it